loveship/components/inputs/TextInput.tsx

```diff
@@ -1,24 +1,24 @@
 import * as React from 'react';
-import * as types from '../types';
+import * as types from '@epam/uui';
+import { IEditableDebouncerOptions, withMods } from '@epam/uui-core';
+import { TextInput as uuiTextInput, SearchInput as UuiSearchInput } from '@epam/uui';
+import { TextInputProps } from '@epam/uui-components';
+import { TextSettings } from '../../helpers/textLayout';
+import { systemIcons } from '../icons/icons';
 import * as css from './TextInput.scss';
 import * as colorStyle from '../../assets/styles/scss/loveship-color-vars.scss';
-import { withMods, IEditableDebouncer, IEditableDebouncerOptions } from '@epam/uui-core';
-import { TextInput as uuiTextInput, TextInputProps } from '@epam/uui-components';
-import { getTextClasses, TextSettings } from '../../helpers/textLayout';
-import { systemIcons } from '../icons/icons';
 
 const defaultSize = '36';
 
-export interface TextInputMods extends types.EditMode, TextSettings {
+export interface TextInputMods extends types.IHasEditMode, TextSettings {
```

Comment:
why we need here extends types.IHasEditMode, TextSettings?

Reply:
done